includes/admin/class-wp-job-manager-cpt.php

```diff
@@ -303,6 +305,84 @@ public function jobs_by_category() {
 		echo $output;
 	}
 
+	/**
+	 * Output dropdowns for filters based on post meta.
+	 *
+	 * @since 1.31.0
+	 */
+	public function jobs_meta_filters() {
+		global $typenow;
+
+		// Only add the filters for job_listings
+		if ( 'job_listing' !== $typenow  ) {
+			return;
+		}
+
+		// Filter by Filled.
+		$this->jobs_filter_dropdown( 'job_listing_filled', array(
+			array(
+				'value' => '',
+				'text'  => 'Select Filled',
```

Comment:
Wrap these strings in `__( '~~string~~', 'wp-job-manager' )`

Reply:
Good catch. Fixed.